modules/nf-core/survivor/bedpetovcf/main.nf

```diff
@@ -0,0 +1,45 @@
+
+process SURVIVOR_BEDPETOVCF {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/survivor:1.0.7--h9a82719_1':
+        'biocontainers/survivor:1.0.7--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(bedpe)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
```

Comment:
Does the tool take any optional arguments at all? As this is currently missing the args that virtually all modules have

Reply:
This tool does not have any arguments.